internal/view/pod.go

```diff
@@ -242,8 +242,8 @@ func resumeShellIn(a *App, c model.Component, path, co string) {
 }
 
 func shellIn(a *App, path, co string) {
-	args := computeShellArgs(path, co, a.Conn().Config().Flags().KubeConfig)
-
+	os := getPodOS(a.factory, path)
```

Comment:
@galsolom getPodOS should return string, error in case we can't locate the node selector. We can log it here at the call site vs buried in the call.